test/miner_payment_v2_txn_SUITE.erl

```diff
@@ -1,3 +1,4 @@
+%%% RELOC MOVE - not miner-specific functionality
```

Comment:
This looks redundant to existing payment tests in core. Maybe just remove.

Reply:
I think that both payments tests can be removed, yes.